extensions/vscode/src/utils/webviewConduit.ts

```diff
@@ -63,14 +63,11 @@ export class WebviewConduit {
   };
 
   public sendMsg = (msg: HostToWebviewMessage) => {
-    const e = JSON.stringify(msg);
+    // don't send messages if the Webview hasn't initialized yet
```

Comment:
The change here was an optimizatino as well, there is no reason to send the message, the webview will query what it needs when it is initialized.